File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/42_compito_python_20230116/GiaccaAVento.py
from Articolo import Articolo


class GiaccaAVento(Articolo):
    def __init__(self, codice, tipo, marca, taglia, modello, costo, tasche, sconto):
        super().__init__(codice, tipo, marca, taglia, modello, costo, sconto)
        self._tasche = tasche

    def __str__(self):
        return super().__str__() + "\t" + str(self._tasche) + "\t-\t" + str(self._costo) + "\t" + str(self._sconto)

File: Programming-languages-and-laboratory/Java/Exercises/42_compito_python_20230116/GiaccaAVento.java
public class GiaccaAVento extends Articolo {
    private int tasche;

    public GiaccaAVento(int codice, String tipo, String marca, String taglia, String modello, float costo, int tasche,
            int sconto_in_percentuale) {
        super(codice, tipo, marca, taglia, modello, costo, sconto_in_percentuale);
        this.tasche = tasche;
    }

    public String toString() {
        return super.toString() + "\t" + tasche + "\t-\t" + costo + "\t" + sconto_in_percentuale;
    }
}